packages/codemods/transforms/v4/icon/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+const CODEMOD_ANALYSIS = process.env.CODEMOD_ANALYSIS;
+
+module.exports = function ({ source /*, path*/ }, { parse, visit }) {
+  const ast = parse(source);
+
+  const { preserveLayout } = JSON.parse(process.env.CODEMOD_CLI_ARGS);
+
+  return visit(ast, (env) => {
+    let { builders: b } = env.syntax;
+
+    return {
+      ElementNode(node) {
+        if (node.tag === 'FlightIcon') {
+          // filter out the `@isInlineBlock` attribute
+          const outputAttrs = node.attributes.filter((a) => a.name !== '@isInlineBlock');
+
+          // look up for `@isInlineBlock`
+          const attr = node.attributes.find((a) => a.name === '@isInlineBlock');
+
+          // @isInlineBlock attr has been set on the element
+          if (attr) {
+            const isHandlebarsAttr = attr.value.type === 'MustacheStatement';
+
+            if (isHandlebarsAttr && attr.value.path.original !== false) {
+              // rename attribute as @isInline, keep value
+              const updatedAttr = b.attr('@isInline', attr.value);
+              outputAttrs.push(updatedAttr);
+            }
+
+            // if the value is false or not handlebars, we don't need to add the attribute
```

Comment:
Is this comment still correct? 🤔

Reply:
But it's above the `} else if (preserveLayout)` so I interpret the comment as related to that line, which instead is doing something else. No>